src/components/photopage/profile.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+
+const cropperWidth = '200px';
+const styles = theme => ({
+	container: {
+		display: 'flex',
+		flexDirection: 'column',
+		justifyContent: 'center',
+		alignItems: 'center',
+		padding: theme.spacing.unit * 2.75
+	},
+	// this cropper only crops landscape-oriented/square image
+	cropper: {
+		width: cropperWidth,
+		height: cropperWidth,
+		borderRadius: '50%',
+		overflow: 'hidden',
+		display: 'flex',
+		justifyContent: 'center'
+	},
+	photo: {
+		margin: '0 auto',
+		display: 'inline',
+		height: '100%',
+		width: 'auto'
+	},
+	name: {
+		fontFamily: ['Chivo', 'san-serif'],
+		fontSize: theme.typography.fontSize * 1.5,
+		fontWeight: 'normal',
+		margin: theme.spacing.unit,
+		// prevent long names from wrapping
+		whiteSpace: 'nowrap'
```

Comment:
Hmm this seems to be a sign that we want to in fact increase the width. In the worst case this could look pretty bad.